tests/workbenches/notebook-controller/test_spawning.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import pytest
+
+from timeout_sampler import TimeoutExpiredError
+
+from ocp_resources.pod import Pod
+
+
+class TestNotebook:
+    @pytest.mark.parametrize(
+        "unprivileged_model_namespace,users_persistent_volume_claim,default_notebook",
+        [
+            pytest.param(
+                {
+                    "name": "test-odh-notebook",
+                    "dashboard-label": True,
+                },
+                {"name": "test-odh-notebook"},
+                {
+                    "namespace": "test-odh-notebook",
+                    "name": "test-odh-notebook",
+                },
+            )
+        ],
+        indirect=True,
+    )
+    def test_create_simple_notebook(
+        self,
+        unprivileged_client,
+        unprivileged_model_namespace,
+        users_persistent_volume_claim,
+        default_notebook,
+    ):
+        """
+        Create simple Notebook with all needed resources and see if Operator creates it properly
+        """
+        with default_notebook:
```

Comment:
sorry, missed this. not needed as `default_notebook` yields the object which will be cleaned up on teardown

Reply:
fixed, thanks